specs/Deployment/DeploymentAPI.md

```diff
@@ -155,6 +202,15 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         HRESULT ExtendedError{ get; };
     };
 
+    /// This object is used to specify deployment options to apply when using DeploymentManager's
+    /// Initialize method
+    runtimeClass DeploymentOptions
```

Comment:
`DeploymentInitializeOptions`

`DeploymentOptions` is too broad a term. If/when we add another function to DeploymentManager to, say, uninstall, we'd want a different options object with uninstall-specific options e.g. DeploymentUninstallOptions.

`Windows.Management.Deployment.PackageManager.DeploymentOptions` is a good example of an old way we did things ages back and is now an anti-pattern. One type with lots of options for varying operations forming overlapping supersets and subsets (very Venn diagram-y). The recommend pattern nowadays is different types for different uses .eg. `Windows.Management.Deployment.PackageManager.AddPackageOptions` vs `StagePackageOptions` vs `RegisterPackageOptions` for `AddPackageByUriAsync()` vs `StagePackageByUriAsync()` vs `RegisterPackageByUriAsync()`.